client/signup/jetpack-connect/plans-landing.jsx

```diff
@@ -58,7 +56,7 @@ const PlansLanding = React.createClass( {
 			</div>
 		);
 	}
-} );
+}
 
 export default connect(
 	() => {
```

Comment:
The mapDispatchToProps declaration can probably simplified:

From a function:
```js harmony
	( dispatch ) => {
		return Object.assign( {},
			bindActionCreators( { selectPlanInAdvance }, dispatch ),
			{
				recordTracksEvent( eventName, props ) {
					dispatch( recordTracksEvent( eventName, props ) );
				}
			}
		);
	}
```

to a plain object
```
{
  selectPlanInAdvance,
  recordTracksEvent
}

```